docs/azure-tre-features/airlock.md

```diff
@@ -0,0 +1,130 @@
+# Airlock
+
+In a Trusted Research Environment (TRE) the workspaces represent a security boundary that enables researchers to access data, execute analysis, apply algorithms and collect reports. The airlock capability brings the only mechanisme that allows users to `import` or `export` data and tools in a secure fashion with a manual approval.
+This constitutes the mechanism focused in preventing malware inside TRE and/or TRE workspaces and preventing data exfiltration, while allowing researchers to execute their tasks.
+The airlock feature brings several actions: ingress/egress Mechanism; Data movement; Security gates; Approval mechanism and Notifications. As part of TRE's Safe settings all activity must be tracked for auditiing purpose.
```

Comment:
Can we have a flow diagram? I'd move this below the bullets below.

Reply:
Added a state diagram only. It will be useful next to the process description.
Down below we can add one focusing on the data movement through the storage accounts and respective security boundaries. Makes sense?